stripper/ze_666_crazy_escape_v2_3.cfg

```diff
@@ -0,0 +1,26 @@
+;Lock admin level change buttons above spawn, as a zombie can be boosted to one and press it
+modify:
+{
+	match:
+	{
+		"classname" "func_button"
+		"targetname" "nrk_stage1_button"
+	}
+	replace:
+	{
+		"spawnflags" "3073"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "func_button"
+		"targetname" "nrk_stage2_button"
+	}
+	replace:
+	{
+		"spawnflags" "3073"
+	}
+}
```

Comment:
Surrounding the admin room with brushes would be the better solution here. However it's probably worth waiting for Naraka's response before jumping to our own stripper for this.

Reply:
Yeah naraka has already released v2_4 with this issue fixed https://gamebanana.com/mods/79591